dcos_installer/action_lib.py

```diff
@@ -379,7 +379,24 @@ def _add_prereqs_script(chain):
 distro="$(source /etc/os-release && echo "${ID}")"
 if [[ "${distro}" == 'coreos' ]]; then
   echo "Distro: CoreOS"
-  echo "CoreOS includes all prerequisites by default." >&2
+  sudo systemctl is-enabled systemd-networkd > /dev/null 2>&1
```

Comment:
What is this doing?

Reply:
Oh... yeah this is super confusing bash. You can just do this line in the if statement rather than capturing the exit code and doing string comparison...